app/views/articles/show.html.erb

```diff
@@ -30,13 +30,16 @@
       <%= heading_tag article.title %>
 
       <% unless @newsletter_excluded %>
-        <%= render 'shared/news_signup_test',
-                   ab_test_render: true %>
+        <%= render 'shared/news_signup_test', ab_test_render: true %>
         <%= render 'shared/news_signup_inpage', ab_test_render: false %>
       <% end %>
 
       <%= article.content %>
 
+      <% unless @newsletter_excluded %>
+        <%= render 'inline_newsletter_sign_up' %>
```

Comment:
Maybe we should rename this partial to `inline_newsletter_signup` to be consistent with the naming of the shared partials above?